tests/test_interface.rb

```diff
@@ -958,6 +959,44 @@ def test_interface_ipv4_arp_timeout
     assert_raises(RuntimeError) { nonvlanint.ipv4_arp_timeout = 300 }
   end
 
+  def test_interface_fabric_forwarding_anycast_gateway
+    # Setup
+    config('no interface vlan11')
+    int = Interface.new('vlan11')
+    config('feature fabric forwarding')
+    foo = VxlanGlobal.new
+    foo.anycast_gateway_mac = '1223.3445.5668'
```

Comment:
@smigopal I spoke with @glennmatthews and he clarified that his point is that `OverlayGlobal.new.anycast_gateway_mac` will handle turning `feature fabric forwarding` on `I2` images and ignore cli failures on `I3` images.